ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-stable-prod.yaml

```diff
@@ -68,6 +68,38 @@ tests:
     test:
     - ref: openshift-qe-ingress-perf
     workflow: rosa-aws-sts
+- always_run: false
+  as: perfscale-rosa-singleaz-network-perf-2nodes
+  steps:
+    cluster_profile: aws-qe
+    env:
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ENABLE_AUTOSCALING: "false"
+      MULTI_AZ: "false"
+      OPENSHIFT_VERSION: "4.12"
+      REGION: us-west-2
+      REPLICAS: "2"
+      ROSA_LOGIN_ENV: production
+    test:
+    - chain: openshift-qe-network-perf
+    workflow: rosa-aws-sts
+- as: perfscale-rosa-hcp-1zone-network-perf-2nodes-periodic
+  cron: 0 7 * * 1
```

Comment:
@jtaleric Do we want to run this in production on a cron basis for 4.12? Just want to make sure that it was not added here as periodic job accidentally.